src/test/java/uk/gov/hmcts/cmc/claimstore/events/OfferRespondedCitizenActionsHandlerTest.java

```diff
@@ -0,0 +1,119 @@
+import uk.gov.hmcts.cmc.claimstore.config.properties.notifications.NotificationsProperties;
+import uk.gov.hmcts.cmc.claimstore.controllers.utils.sampledata.SampleClaim;
+import uk.gov.hmcts.cmc.claimstore.models.Claim;
+import uk.gov.hmcts.cmc.claimstore.models.offers.MadeBy;
+import uk.gov.hmcts.cmc.claimstore.models.offers.Settlement;
+import uk.gov.hmcts.cmc.claimstore.services.notifications.OfferMadeNotificationService;
+import uk.gov.service.notify.NotificationClientException;
+
+import static org.mockito.ArgumentMatchers.anyMap;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class OfferRespondedCitizenActionsHandlerTest {
+
+    private static final String FRONTEND_URL = "domain";
+    private static final String OFFER_ACCEPTED_TO_DEFENDANT = "offer accepted, email to defendant";
+    private static final String OFFER_ACCEPTED_TO_CLAIMANT = "offer accepted, email to claimant";
+    private static final String OFFER_REJECTED_TO_DEFENDANT = "offer rejected, email to defendant";
+    private static final String OFFER_REJECTED_TO_CLAIMANT = "offer rejected, email to claimant";
+    private static final Claim claimWithOffer = SampleClaim.builder().withSettlement(mock(Settlement.class)).build();
+
+    private OfferRespondedCitizenActionsHandler handler;
+
+    @Mock
+    private OfferMadeNotificationService offerMadeNotificationService;
+
+    @Mock
+    private NotificationsProperties notificationsProperties;
+    @Mock
+    private NotificationTemplates templates;
+    @Mock
+    private EmailTemplates emailTemplates;
+
+    @Before
+    public void setup() {
+        when(notificationsProperties.getTemplates()).thenReturn(templates);
+        when(notificationsProperties.getFrontendBaseUrl()).thenReturn(FRONTEND_URL);
+        when(templates.getEmail()).thenReturn(emailTemplates);
+        when(emailTemplates.getOfferAcceptedByClaimantEmailToDefendant()).thenReturn(OFFER_ACCEPTED_TO_DEFENDANT);
+        when(emailTemplates.getOfferAcceptedByClaimantEmailToClaimant()).thenReturn(OFFER_ACCEPTED_TO_CLAIMANT);
+        when(emailTemplates.getOfferRejectedByClaimantEmailToDefendant()).thenReturn(OFFER_REJECTED_TO_DEFENDANT);
+        when(emailTemplates.getOfferRejectedByClaimantEmailToClaimant()).thenReturn(OFFER_REJECTED_TO_CLAIMANT);
+
+
+        handler = new OfferRespondedCitizenActionsHandler(offerMadeNotificationService, notificationsProperties);
+    }
+
+    @Test
+    public void shouldSendNotificationsToClaimantWhenOfferAccepted() throws NotificationClientException {
+
+        OfferAcceptedEvent event = new OfferAcceptedEvent(claimWithOffer, MadeBy.CLAIMANT);
+
+        handler.sendNotificationToClaimantOnOfferAcceptedByClaimant(event);
+
+        verify(offerMadeNotificationService).sendNotificationEmail(
```

Comment:
I would consider having that type of test in the claim-store integration tests to make sure notification is triggered on endpoint call.

Reply:
I have this test in int tests :)

Reply:
Please see `shouldAcceptOfferAndSendNotifications` and `shouldRejectOfferAndSendNotifications`